### ISSUE TITLE
panel.js changed() method does not resize the editor

### ISSUE THREAD

**Issue by josiahdaniels** [codemirror/CodeMirror#4683]
The Panel.prototype.changed method does not resize the main editor. This is because 'info.height' is `undefined` so `cm._setSize` is being called with `NaN` as the height.

I believe this is the correct fix:

```javascript

  Panel.prototype.changed = function(height) {
    var newHeight = height == null ? this.node.offsetHeight : height;
    var info = this.cm.state.panels;

    this.cm._setSize(null, info.height += (newHeight - this.height)); // OLD
    this.cm._setSize(null, info.heightLeft -= (newHeight - this.height)); // NEW

    this.height = newHeight;
  };
```

**Comment by marijnh** [comment]
You are right, I think. I've made the change you suggested in fd052625.

**Comment by sciamannikoo** [comment]
If I'm seeing right, this commit is included in 5.25.2, but the issue is still present.
